src/interface/button.js

```diff
@@ -32,6 +33,10 @@ export const CardForm : LazyProtectedExport<CardFormComponent> = {
     __get__: () => protectedExport(getCardFormComponent())
 };
 
+export const PaymentFields : LazyProtectedExport<PaymentFieldsComponent> = {
+    __get__: () => getPaymentFieldsComponent()
```

Comment:
Should we add the domain check `protectedExport` wrapper around `getPaymentFieldsComponent`? All other components have it.
By adding protectedExport we were getting this error sometimes
<img width="927" alt="image" src="https://user-images.githubusercontent.com/22331155/170097869-72fc52ff-eee5-432b-bede-c86c41ec5073.png">

Reply:
The `protectedExport` means it is not added to the `window.paypal.Buttons` global object.

Reply:
@mnicpt Thanks for the clarification! 
So in that case how are we able to access all these components like `CardForm`, `Checkout`, `Menu` etc [here in spb](https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/button/props.js#L123) from `window.paypal` global object?
We are trying to access `PaymentFields` similar fashion [here](https://github.com/paypal/paypal-smart-payment-buttons/pull/405/files#diff-e3a4efeee1834a0c5a8750c378aef4deb5d1a4d1638b72565c8b91759ab23211R124)